MAINTAINERS.md

```diff
@@ -7,4 +7,5 @@ This file lists the maintainers of the Stanza project. The responsibilities of m
 | ---- | --------- | ----------- |
 | [Mike Kelly](mailto:mike.kelly@bluemedora.com) | [mkelly](https://github.com/mkelly) | observIQ |
 | [Dan Jaglowski](mailto:dan.jaglowski@bluemedora.com) | [djaglowski](https://github.com/djaglowski) | observIQ |
-| [Camden Cheek](mailto:camden.cheek@bluemedora.com) | [camdencheek](https://github.com/camdencheek) | observIQ |
\ No newline at end of file
+| [Camden Cheek](mailto:camden.cheek@bluemedora.com) | [camdencheek](https://github.com/camdencheek) | observIQ |
```

Comment:
Should these use our `@observiq.com` emails?

Reply:
I'm going to leave these for now. I think each maintainer should update their own as they like.